### PR TITLE
update

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -56,3 +45,52 @@ export default class NetworkLayer extends Relay.DefaultNetworkLayer {
...
+            .join('\n');
+        })
+        .join('\n');
+
+      return `${prefix + message}\n${l}`;
+    })
+    .join('\n');
+}
+
+function createRequestError(request, responseStatus, payload) {
```
Comment (taion) [inline src/index.js]: any chance we could move these both into `__forks__`? just to make it clear they're copied verbatim (unless they aren't)

File: src/index.js [hunk 2]
```diff
@@ -1,43 +1,32 @@
...
+      .then(payloads => {
+        requests.forEach((request, index) => {
+          let payload;
+          if (payloads && payloads[index]) {
+            payload = payloads[index];
+          } else {
+            payload = payloads;
+          }
+
+          if ('errors' in payload) {
```
Comment (taion) [inline src/index.js]: is this the only functional change here? what was wrong w/the `hasOwnProperty` check?

**Comment by taion** [comment]
@jquense I updated this a bit more and re-synced the fork in https://github.com/taion/relay-network-layer/pull/2/commits/7715202be7c977473cbd2290f58d27dc88274d17. Can you take a quick look?

**Comment by jquense** [comment]
LGTM

**Comment by taion** [comment]
How do we test this BTW?

**Comment by taion** [comment]
Oh, I guess I have, like, a bunch of Relay example repos and all.

**Comment by jquense** [comment]
use it and see if it works? I copied from web and it was working there so prolly a good bet?

**Comment by jquense** [comment]
lol

**Comment by taion** [comment]
v0.2.0

**Pull request merged**